nixos/tests/davis.nix

```
@@ -0,0 +1,57 @@
+import ./make-test-python.nix (
+  { lib, pkgs, ... }:
+
+  {
+    name = "davis";
+
+    meta.maintainers = with lib.maintainers; [ ];
```

```suggestion
    meta.maintainers = with lib.maintainers; [ ];
```
This could be in sync with package that this is testing: `davis.meta.maintainers`
If someone only wants to not follow a certain file, it is possible to filter them out of it. This way, a person that is interested in package, only needs to add themselves once, and follow the tests and module. I think this pattern better. While, the optionality exist.